airbyte-integrations/connectors/source-github/source_github/source.py

```diff
@@ -54,21 +55,19 @@
 
 class SourceGithub(AbstractSource):
     def _generate_repositories(self, config: Mapping[str, Any], authenticator: TokenAuthenticator) -> List[str]:
-        organizations = list(filter(None, config["organization"].split(" ")))
         repositories = list(filter(None, config["repository"].split(" ")))
 
-        if not (organizations or repositories):
-            raise Exception("Either `organisation` or `repository` need to be provided for connect to Github API")
+        if not repositories:
+            raise Exception("Field `repository` required to be provided for connect to Github API")
 
-        repositories_list = []
+        repositories_list = [repo for repo in repositories if not re.match("^.*/\\*$", repo)]
+        organizations = [org.split("/")[0] for org in repositories if org not in repositories_list]
```

Comment:
Are we going to pull all repos from organization without the user request this? @yevhenii-ldv

Reply:
@marcosmarxm If the user specifies, for example, `airbytehq/*`, this means that he wants to pull all the repositories for this organization.

Reply:
btw, there is no unit_test for this? would be amazing to have a test mocking this function.